ISSUE_TEMPLATE.md

```diff
@@ -1,39 +1,60 @@
-Please try to adhere to the guidelines below as much as possible when
-submitting your issue.
-- Verify that your issue is not being currently addressed by other
-[issues](https://github.com/scikit-learn/scikit-learn/issues?q=)
-or [pull requests](https://github.com/scikit-learn/scikit-learn/pulls?q=).
-- If your issue is a usage question or does not potentially require
-changes to the codebase to be solved, then
-[StackOverflow](http://stackoverflow.com/questions/tagged/scikit-learn)
-(using the`[scikit-learn]` tag) or our
-[mailing list](https://lists.sourceforge.net/lists/listinfo/scikit-learn-general)
-may be a better place to bring it up. For more information, see
-[User Questions](http://scikit-learn.org/stable/support.html#user-questions).
-
-If you are submitting a bug issue:
-- Please include your operating system type and version number, as well
-as your Python, scikit-learn, numpy, and scipy versions. This information
-can be found by runnning the following code snippet:
+<!--
+If your issue is a usage question or does not potentially require
+changes to the codebase to be solved, submit it here instead:
+- StackOverflow with the scikit-learn tag (http://stackoverflow.com/questions/tagged/scikit-learn)
```

Comment:
I would remove the parentheses around the link, it increases the likelihood of the copying and pasting to go wrong.